tests/hotplug/BUILD.bazel

```diff
@@ -16,7 +16,6 @@ go_library(
         "//pkg/apimachinery/patch:go_default_library",
         "//pkg/libvmi:go_default_library",
         "//pkg/pointer:go_default_library",
-        "//pkg/virt-launcher/virtwrap/api:go_default_library",
```

Comment:
Commit message: Could you please add the motivation behind this change?

Reply:
Yes, done